mylist.cpp

```
@@ -252,12 +368,13 @@ listnode mylist::front() const
 //
 // return the last node in the list
 //
-listnode mylist::back() const
+listnode mylist::back() const // my part
 {
-  // change me, your code here.
-  cout << "back is not supported.\n";
-  return listnode();
-
+  if (tail == NULL) {
+    cout << "back: can't get back from an empty list.\n";
+    exit(0);
```

Never use `exit(0)`. You can throw an exception here.